x-pack/legacy/plugins/ml/public/application/jobs/new_job/common/job_validator/validators.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { distinctUntilChanged, filter, map, switchMap } from 'rxjs/operators';
+import { Observable, Subject } from 'rxjs';
+import {
+  CardinalityModelPlotHigh,
+  CardinalityValidationResult,
+  ml,
+} from '../../../../services/ml_api_service';
+import { JobCreator } from '../job_creator';
+
+export enum ValidatorSeverity {
```

Comment:
looks like this doesn't need to be a string enum. also in our informal naming convention we use caps  for constant enums like this